partner_firstname/__openerp__.py

```diff
@@ -15,35 +17,34 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
 {
-    'name': 'Partner first name, last name',
+    'name': 'Partner first name and last name',
     'description': """
-This module splits first name and last name for non company partners
-====================================================================
+Split first name and last name for non company partners
```

Comment:
Can you move description under README.rst ? using OCA template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst